sup3r/pipeline/forward_pass.py

```diff
@@ -786,23 +798,26 @@ def file_ids(self):
             List of file ids for each output file. Will be used to name output
             files of the form filename_{file_id}.ext
         """
-        if self._file_ids is None:
+        if not self._file_ids:
             n_chunks = len(self.time_index)
             n_chunks /= self.fwp_chunk_shape[2]
             n_chunks = np.int(np.ceil(n_chunks))
             self._file_ids = []
             for i in range(n_chunks):
```

Comment:
We should rename n_chunks if this is now only the temporal chunks. Otherwise i would assume n_chunks is the total number of temporal chunks * spatial chunks (which it should be).